content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- fabriziopandini
+- luxas
+- timothysc 
+title: Customizing control plane flags with kubeadm
+---
+
+{{% capture overview %}}
+
+kubeadm’s configuration exposes the following fields that can be used to override the default flags passed to control plane components such as the APISever, ControllerManager and Scheduler:
+
+- `APIServerExtraArgs`
+- `ControllerManagerExtraArgs`
+- `SchedulerExtraArgs`
+
+The fields can contain a list of `key: value` pairs. To override flags for a specific control plane component just include one of the fields above and start adding flags for it.
+
+{{< note >}}
+In later versions of the kubeadm configuration these fields will be replaced by usage of a ComponentConfig and a ConfigMap.
+{{< /note >}}
+
+{{% /capture %}}
+
+## APIServer flags
+
+The list of possible APIServer flags and their descriptions can be found [here](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
+
+Example usage:
+```yaml
+apiServerExtraArgs:
+  advertise-address: 192.168.0.103
+  allow-privileged: true
```

Comment:
is deprecated I think... some other common value like `enable-admission-control`?

Reply:
The most common uses are:

adding admission controllers 
configuring audit log (soon to be webhook)